src/mame/drivers/paranoia.cpp

```diff
@@ -92,6 +93,11 @@ void paranoia_state::pce_io(address_map &map)
 	map(0x00, 0x03).rw("huc6270", FUNC(huc6270_device::read), FUNC(huc6270_device::write));
 }
 
+void paranoia_state::vdc_mem(address_map &map)
```

Comment:
At this point `vdc_mem` can totally be derived from `pce_common_state` as protected function, no need to repeat memory maps in clients.